pango/src/layout.rs

```diff
@@ -48,6 +48,21 @@ impl LayoutLine {
         unsafe { (*self.to_glib_none().0).length }
     }
 
+    // rustdoc-stripper-ignore-next
+    /// Returns `true` if this is the first line of the paragraph.
+    pub fn is_paragraph_start(&self) -> bool {
+        let field = unsafe { (*self.to_glib_none().0).is_paragraph_start };
+        (field >> 24) != 0 // We only keep the bigger byte to get "is_paragraph".
```

Comment:
It's one bit, not one byte. And AFAIU it's implementation defined where exactly that would be located (might be different per platform, endianness, toolchain and especially MSVC vs. gcc/clang do things differently)